### PR TITLE
Explore api with new fields

### PR DESCRIPTION
**Pull request by ethervoid**
* [X] Fixed error while trying to get the synchronization using visualization_id
* [X] Added synced field
* [X] Added table names field
* [X] Added new view to act as the public endpoint (**we have to remove permissions to the private table and give them to this view**)

### CODE REVIEW

**Comment by ethervoid** [comment]
restest this please

**Comment by ethervoid** [comment]
Please @juanignaciosl check this CR

File: lib/tasks/explore_api.rake [hunk 1]
```diff
@@ -49,9 +78,15 @@ namespace :cartodb do
...
 
+    task :setup_public_view => [:environment] do
+      user = target_user
+      user.in_database.run CREATE_PUBLIC_VIEW
+    end
+
     desc "Deletes the #{VISUALIZATIONS_TABLE} table"
     task :drop => [:environment] do
       target_user.in_database.run DROP_TABLE_SQL
+      target_user.in_database.run DROP_PUBLIC_VIEW_SQL
```
Comment (juanignaciosl) [inline lib/tasks/explore_api.rake]: Does DROP TABLE works before dropping the view?

**Comment by juanignaciosl** [comment]
:+1:

File: lib/tasks/explore_api.rake (hunk 1 above)
Comment (ethervoid) [inline lib/tasks/explore_api.rake]: Nope, I've tested it and dropping first one or another always need cascade (a views is a SQL to the table but have insert, update and delete privileges) so for me the cascade should go in the table itself

**Pull request merged**